client/components/search/index.jsx

```diff
@@ -271,19 +280,26 @@ const Search = React.createClass( {
 			spellCheck: 'false'
 		};
 
-		searchClass = classNames( this.props.additionalClasses, {
+		const searchClass = classNames( this.props.additionalClasses, this.props.dir, {
 			'is-expanded-to-container': this.props.fitsContainer,
 			'is-open': isOpenUnpinnedOrQueried,
 			'is-searching': this.props.searching,
-			rtl: this.props.dir === 'rtl',
-			ltr: this.props.dir === 'ltr',
+			'no-close-button': this.props.hideClose,
```

Comment:
Yes, either is fine, but I remember not needing such a class because we could infer it from others. (There's already a ton of classes and props being passed to this component, and I think `isOpenUnpinnedOrQueried` was handling much of it.)